### PR TITLE
Workshop

### PR DESCRIPTION
**Pull request by MakisH**
I added a few more information for the workshop:
* Link to the registration form
* Call for contributions
* Note to avoid flying
* Recommended hotel skeleton text

Will not be merged until we have all the missing information.

Firefox_Screenshot_2019-10-07T11-39-58 730Z

### CODE REVIEW

File: 09-workshop.md [hunk 1]
```diff
@@ -37,18 +37,47 @@ If you are in Munich the day before, join us for a social event on Sunday. (opti
...
+*  Multi-scale coupling
+*  Building, packaging, testing, documentation and other Research Software Engineering topics
+
+Please use the [registration form](https://www5.in.tum.de/workshops/precice-workshop/) if you would like to submit a talk.
 
 ## Getting there
 
 The TUM campus Garching is well connected to the Munich airport (MUC), as well as to the city center and central station. You can reach it using the subway line U6 (station Garching-Forschungszentrum). Have a look at the [Directions page of TUM](https://www.tum.de/en/about-tum/contact-directions/) for more.
 
+We would be very happy if you could choose a train or other environmentally-friendly ways to come to Munich. Please avoid flying!
```
Comment (uekerman) [inline 09-workshop.md]:
```suggestion
We would be very happy if you could choose a train or other environmentally-friendly ways to come to Munich. Please avoid flying if possible.
```

File: 09-workshop.md [hunk 2]
```diff
@@ -37,18 +37,47 @@ If you are in Munich the day before, join us for a social event on Sunday. (opti
...
 ## Accommodation
 
-We are preparing a list of recommended hotels, which will be available later this year and announced to our [mailing list](https://mailman.informatik.uni-stuttgart.de/mailman/listinfo/precice).
+We have reserved a limited number of rooms for you at the hotel (TBA) for the price of (TBA) per night, including breakfast. We recommend this offer, as this very good, bavarian-style hotel is very close to the workshop venue and to the restaurant where we will host the invited dinner. It is also very well connected to the city center.
+
+Please use the code (TBA) when booking.
+
+## Important dates
+
+* September 27: Registration opens.
```
Comment (uekerman) [inline 09-workshop.md]: Let's update this before merging.

**Review by uekerman — CHANGES_REQUESTED** [review]

File: 09-workshop.md (hunk 2 above)
Comment (MakisH) [reply 09-workshop.md]: I removed it for now.

File: 09-workshop.md [hunk 3]
```diff
@@ -23,32 +23,74 @@ The following is a rough preliminary schedule for the workshop. The main part of
...
+* 15:30 - 16:30 Coffee Break, time for individual discussions
+* 16:30 - 17:30 Developer Presentations (2x) 
+* 17:30 - 18:30 Contributed Talks (3x)
+* 19:30 Dinner
 
 ### Tuesday, February 18
-* 9:00 - 12:00 More presentations by users, introduction to new features, invited presentations, and discussions in long coffee breaks
+* 09:00 - 10:00 Contributed Talks (3x)
+* 10:00 - 10:30 Developer Presentation (1x)
+* 10:20 - 11:00 Coffee Break
```
Comment (uekerman) [inline 09-workshop.md]:
```suggestion
* 10:30 - 11:00 Coffee Break
```

File: 09-workshop.md [hunk 4]
```diff
@@ -23,32 +23,74 @@ The following is a rough preliminary schedule for the workshop. The main part of
...
+
+Please use the code `preCICE` when [booking](https://www.hoyackerhof.de/en/booking/). The offer is available until January 7.
+
+## Funding and travel grants
+
+This is a no-profit, academic conference. A big "thank you" to [SPPEXA](http://www.sppexa.de/) for the financial support of the Workshop! With this support, we are able to provide a small number of travel grants to (PhD) students who would not be able to join otherwise. If this applies to you, please explain your situation in the registration form.
+
+## Important dates
+
+* November 15: End or early registration, abstract submission deadline, travel grant application deadline.
```
Comment (uekerman) [inline 09-workshop.md]:
```suggestion
* November 15: End of early registration, abstract submission deadline, travel grant application deadline.
```

**Review by uekerman — APPROVED** [review]

**Pull request merged**